README.md

```diff
@@ -63,7 +65,85 @@ teracy-dev:
       path:
         lookup: workspace
       location:
-        git: git@github.com:hoatle/teracy-dev-core.git # your forked repo
+        git:
+          remote:
+            origin: <fill your forked repo here>
+            upstream: https://github.com/teracyhq-incubator/teracy-dev-core.git
         branch: develop
       require_version: ">= 0.5.0-SNAPSHOT"
 ```
+
+- Run `vagrant up` or `vagrant reload --provision` (if your vagrant machine is running) to apply your configure.
+
+## Supported configuration
+These are the supported configuration in teracy-dev-core, you can configure in `workspace/teracy-dev-entry/config_override.yaml`:
+
+- Variables
+
+  * `variables` is used to define dynamic configuration values for use in configuration files. After setting, you can call them by `%{env_key}`.
+
+  * You can define variable keys with environment variables. The following is the configuration format:
+
+  ```yaml
+    variables:
+      key: ${ENV_VAR}
+      key: ${ENV_VAR-:default}
+      key: value
+    # and "%{key}" can be used for settings values if available
+  ```
+
+  * [Example for variables](https://github.com/teracyhq-incubator/teracy-dev-core/tree/develop/docs/examples/variables.yaml)
```

Comment:
use relative links so that when we git tag or change branch, it should point to the docs at that branch/tag.